BrowserKit/Sources/Common/Extensions/StringExtension.swift

```diff
@@ -11,3 +11,15 @@ extension StringProtocol {
         return replacingOccurrences(of: target, with: "")
     }
 }
+
+public extension String {
+    /// Returns a new string made by removing the leading String characters contained
+    /// in a given character set.
+    func stringByTrimmingLeadingCharactersInSet(_ set: CharacterSet) -> String {
+        var trimmed = self
+        while trimmed.rangeOfCharacter(from: set)?.lowerBound == trimmed.startIndex {
+            trimmed.remove(at: trimmed.startIndex)
+        }
+        return trimmed
+    }
```

Comment:
What is the issue you are trying solved? I think for now I just keep the code as is but how about you give a presentation in the iOS Engineering so that the whole team learns?